src/main/ui/app/sass/addonindex.scss

```diff
@@ -26,14 +26,32 @@
   float: right;
 
 }
+.maintable{
+margin-bottom:0;
+}
+
+
+.titletype{
```

Comment:
Minor repositioning of the heading is needed as it is not part of the list unlike a table